client/web/src/search/results/AnnotatedSearchExample.tsx

```diff
@@ -0,0 +1,181 @@
+                width={width}
+                height={height}
+                viewBox={`${viewBoxX} 35 ${width} ${height}`}
+                preserveAspectRatio="xMidYMid meet"
+                fill="none"
+                xmlns="http://www.w3.org/2000/svg"
+            >
+                <g>
+                    <path
+                        d={`M ${inputStart} 113 c 0 -1.381 1.1193 -2.5 2.5 -2.5 H 688.5 v 33 H ${
+                            inputStart + 2.5
+                        } c -1.3807 0 -2.5 -1.119 -2.5 -2.5 v -28 z`}
+                        className={styles.searchBox}
+                    />
+                    {showSearchContext && (
+                        <>
+                            <text className={styles.code} x="68" y="130.836">
+                                <tspan className={styles.filter}>context:</tspan>
+                                <tspan>global</tspan>
+                            </text>
+                            <line className={styles.separator} x1="178" y1="118" x2="178" y2="137" />
+                        </>
+                    )}
+                    <text className={styles.code} x="171.852" y="130.836">
+                        <tspan className={styles.filter}>{'  '}repo:</tspan>
+                        <tspan>sourcegraph/sourcegraph</tspan>
+                        <tspan className={styles.metaRegexpCharacterSet}>.</tspan>
+                        <tspan className={styles.metaRegexpRangeQuantifier}>*</tspan>
+                        <tspan> function auth(){'{'} </tspan>
+                    </text>
+                    <FormatLetterCaseIcon className="icon-inline" x="590" y="115" />
+                    <RegexIcon className="icon-inline" x="620" y="115" />
+                    <CodeBracketsIcon className="icon-inline" x="650" y="115" />
+                    <path
+                        d="M688 110H731C732.105 110 733 110.895 733 112V142C733 143.105 732.105 144 731 144H688V110Z"
+                        fill="#1475CF"
+                    />
+                    <SearchIcon className={classNames(styles.searchIcon, 'icon-inline')} x="698" y="115" />
+
+                    {arrow(188, 30, 'above')}
+                    <text transform={`translate(${filterTextStart}, 44)`}>
+                        <tspan x="0" y="0">
+                            Filters scope your search to repos,{' '}
+                        </tspan>
+                        <tspan x="0" y="16">
+                            orgs, languages, and more.
+                        </tspan>
+                    </text>
+
+                    {arrow(410, 120, 'above')}
+                    <text transform="translate(395, 44)">
+                        <tspan x="0" y="0">
+                            By default, search terms are{' '}
+                        </tspan>
+                        <tspan x="0" y="16">
+                            interpretted literally (without regexp).
```

Comment:
```suggestion
                            interpreted literally (without regexp).
```

Caught this in your latest screenshots 😉

Reply:
Argh....